values.yaml

```
@@ -836,10 +836,16 @@ concourse:
         ##     action: labeldrop
         metricRelabelings:
 
-    ## The value to set for X-Frame-Options. If omitted, the header is not set.
+    ## The value to set for X-Frame-Options. If omitted, the default value in
+    ## the Concourse binary is used.
     ##
     xFrameOptions:
 
+    ## The value to set for Content-Security-Policy header. If omitted, the default
+    ## value in the Concourse binary is used.
+    ##
+    ContentSecuritPolicy:
```

```suggestion
    ContentSecurityPolicy:
```

Also, is it uncommon for the value to be capitalized? Seems like most follow `lowerCamelCase`

---

Can you tell I didn't get enough sleep? 😅